.github/workflows/test-core.yaml

```diff
@@ -0,0 +1,152 @@
+name: Core CI Tests
+on:
```

Comment:
I'm noticing from this PR that we're seeing 2 runs for each test package. One for the push and one for the PR. Is there a way to get this so that we're only running it once?

Reply:
Yeah let's scope this to `push` for now, but I think actually the pull_request is testing the _merge_ commit and the push is testing the _feature_ commit - effectively what we have with Circle, except that does the merge commit run after the merge.